bgpd/bgp_bmp.c

```diff
@@ -746,6 +746,30 @@ static int bmp_peer_backward(struct peer *peer)
 	return 0;
 }
 
+static int bmp_peer_deleted(struct peer *peer)
+{
+	if (peer->status != Deleted) {
+		return 0;
+	}
+
+	zlog_debug("bmp_peer_deleted(%p) host:%s id:%u, doppelganger(%p)", peer,
```

Comment:
Please do not add unguarded debugs to the system.  This will chew up peoples log files ( yeah this is probably rare ).  But please put a `if (XXXX)` to ensure the debug log only happens when we have turned on the appropriate debug

Reply:
add debug option for BMP in new commit